process-engine/process_engine_meta
# 🐛 Fix timer expression parsing

**Changes:**

1. Remove some Model-Parser tests that have become obsolete with the implementation of [this](https://github.com/process-engine/process_engine_core/pull/245).
2. Add tests for checking the TimerFacade's timer validations.

**Issues:**

Part of https://github.com/process-engine/process_engine_runtime/issues/249

PR: #174

## How can others test the changes?

Run the tests. Apart from those that were removed, they should all run as before.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
